go.mod

```diff
@@ -32,10 +32,10 @@ require (
 	google.golang.org/api v0.143.0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.26.4
-	k8s.io/apimachinery v0.26.4
-	k8s.io/client-go v0.26.4
-	sigs.k8s.io/controller-runtime v0.14.6
+	k8s.io/api v0.28.1
+	k8s.io/apimachinery v0.28.1
+	k8s.io/client-go v0.28.1
+	sigs.k8s.io/controller-runtime v0.16.2
```

Comment:
We seem to be still blocked by this [WIP PR in operator-sdk](https://github.com/operator-framework/operator-sdk/pull/6514)

Reply:
@dependabot close